server/database/config/build.sql

```diff
@@ -30,7 +30,8 @@ CREATE TABLE orders_request (
     provider_id INTEGER REFERENCES users(id) ON UPDATE CASCADE NOT NULL,
     description TEXT NOT NULL,
     state VARCHAR(55) DEFAULT 'pending',
-    date TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+    date TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    CONSTRAINT UC_Orders_Request UNIQUE (user_id, provider_id, date)
```

Comment:
since the date has the default value of the time it's inserted, this might differ in the span of milliseconds, so I suggest replacing it with the description because if the same request is sent with the exact body it will throw an error otherwise it's a new request.